tests/stub/configuration_hints/test_connection_recv_timeout_seconds.py

```diff
@@ -72,7 +74,7 @@ def test_timeout(self):
             result = self._session.run("timeout")
             # TODO It will be removed as soon as JS Driver
             # has async iterator api
-            if get_driver_name() in ["javascript"]:
+            if get_driver_name() in ['javascript', 'dotnet']:
```

Comment:
```suggestion
            if get_driver_name() in ["javascript", "dotnet"]:
```

This affects more lines, but I'll only comment once. Please prefer double quotes `"`. Single quotes only, when it is used to avoid escaping double quote characters. In Python, there is no difference between them but I like to have a consistent code base. Makes it for instance easier to search for things.